pkgs/applications/networking/instant-messengers/telegram/telegram-cli/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitHub, jansson, lib, libconfig, libevent, libgcrypt, lua, lua53Packages, openssl, pkg-config, python39, readline, runtimeShell, zlib }:
+
+stdenv.mkDerivation {
+  pname = "telegram-cli";
+  version = "20200106";
+
+  src = fetchFromGitHub {
+    owner = "kenorb-contrib";
+    repo = "tg";
+    rev = "e63c9ddcf391de7f131c5d0849713b2fe14fef11";
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "telegram-cli";
  version = "20200106";

  src = fetchFromGitHub {
    owner = "kenorb-contrib";
    repo = "tg";
    rev = version;
```

Reply:
If I do that, it errors as:

> $ nix build .#telegram-cli
> error: builder for '/nix/store/qb34a2r560cgrnz3zhdbk6ibnlv59ki5-source.drv' failed with exit code 1;
>        last 10 log lines:
>        > remote: Enumerating objects: 87, done.
>        > remote: Counting objects: 100% (87/87), done.
>        > remote: Compressing objects: 100% (72/72), done.
>        > remote: Total 87 (delta 5), reused 39 (delta 3), pack-reused 0
>        > Unpacking objects: 100% (87/87), 180.65 KiB | 635.00 KiB/s, done.
>        > From https://github.com/kenorb-contrib/tg
>        >  * tag               20200106   -> FETCH_HEAD
>        > fatal: Not a valid object name 20200106
>        > Unrecognized git object type:
>        > Unable to checkout 20200106 from https://github.com/kenorb-contrib/tg.git.
>        For full logs, run 'nix log /nix/store/qb34a2r560cgrnz3zhdbk6ibnlv59ki5-source.drv'.
> error: 1 dependencies of derivation '/nix/store/4lbhjglrdy7wdr866jh56r8pvh3lbn02-telegram-cli-20200106.drv' failed to build

Any idea?

Reply:
It works for me.

Reply:
> It works for me.

Maybe you haven't clean-up the sha256. Could you test removing the sha256 field and trying again? It still errors here. And I don't know why...

Reply:
Maybe we need to use ``refs/tags/20200106``?

Reply:
Yes, `refs/tags/20200106` works, I still wonder why `20200106` doesn't.